src/internal.c

```diff
@@ -2388,6 +2388,12 @@ int InitSSL_Ctx(WOLFSSL_CTX* ctx, WOLFSSL_METHOD* method, void* heap)
     ret = wolfEventQueue_Init(&ctx->event_queue);
 #endif /* HAVE_WOLF_EVENT */
 
+#if defined(HAVE_PK_CALLBACKS) && defined(WOLFSSL_MAXQ10XX_TLS)
+    /* Let the maxq10xx infrastructure know whether we are doing TLS 1.3. */
```

Comment:
I would change this comment to "Let the maxq10xx infrastructure know what TLS version we're using."